### PR TITLE
feat: clear CircleCI pnpm job

### PR DESCRIPTION
**Pull request by nferch**
* e2e workflow doesn't appear to be used, but updated anyway
* also: excluded deploy from all scheduled pipelines as it isn't needed for the other "Test Report Daily" schedule

### CODE REVIEW

**Comment by wittjosiah** [comment]
@nferch e2e workflow is used to manually run `e2e` on any branch by setting `e2e` trigger parameter to `true` in the UI

**Comment by nferch** [comment]
Need to update branch in CircleCI to main once this is merged.

File: .circleci/config.yml [hunk 1]
```diff
@@ -218,13 +233,20 @@ workflows:
             - Github
 
   e2e:
-    when: << pipeline.parameters.e2e >>
+    when:
+      and:
+        - << pipeline.parameters.e2e >>
+        - not:
+          equal: [ clear_pnpm_cache, << pipeline.schedule.name >> ]
```
Comment (wittjosiah) [inline .circleci/config.yml:240]: does this need another intent?

**Review by wittjosiah — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (nferch) [reply .circleci/config.yml:240]: I don't think so, it passes the schema validation and didn't trigger on the daily run.

Comment (wittjosiah) [reply .circleci/config.yml:240]: interesting, yaml is confusing sometimes :)

**Pull request merged**